sdk/core/azure-core/DEPENDENCIES.md

```diff
@@ -0,0 +1,419 @@
+# Dependencies
+
+## 1.15.0 (2021-04-02)
+
+<details>
+<summary>Dependencies</summary>
+
+| Dependency | Version |
+| ---------- | ------- |
+| com.fasterxml.jackson.core:jackson-annotations | 2.12.2 |
```

Comment:
I like this :). Are we using any tool for this? Do we want to do it for all our core libraries and not just azure-core?

Also, how do we expect this to be used? Will it be part of the package?

Reply:
There was tooling create to generate this, but this PR is now so old I'll need to find out if it still exists somewhere on my machine.

I would not include this in the JAR shipped to Maven unless we want to make it part of the Javadoc site.

Reply:
Who would be consuming this information? Is this for our own book-keeping then?

Reply:
> Who would be consuming this information? Is this for our own book-keeping then?

For the most part it would be book-keeping (which is why this has gone a bit stale). But, there is some use of it for customer's to eagerly determine whether there will be versioning conflicts with other dependencies they're using.

Reply:
If we expect customers to use it, we should put in the package - no?